server/app/services/docker/streaming_executor.rb

```diff
@@ -63,7 +63,11 @@ def register_stdin_ws_events(shell, tty)
         else
           begin
             input = JSON.parse(event.data)
-            @client.notify('/containers/tty_input', @exec_session['id'], input['stdin']) if input.has_key?('stdin')
+            if input.has_key?('stdin')
+              @client.notify('/containers/tty_input', @exec_session['id'], input['stdin'])
+            elsif input.has_key?('console_size')
```

Comment:
Maybe avoid introducing a new `console` term in the API, and use the `tty` naming for this, like `{ "tty_size": { "width": 80, "height": 24 } }`?

Reply:
Done.